catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -368,7 +367,7 @@ public Metacard transform(InputStream input, String id)
       try (InputStream inputStreamCopy = fileBackedOutputStream.asByteSource().openStream()) {
         extractor = new TikaMetadataExtractor(inputStreamCopy, previewMaxLength, metadataMaxLength);
 
-      } catch (TikaException | RuntimeException t) {
+      } catch (Throwable t) {
```

Comment:
Do we actually want to catch all `Throwable`s here or is `Exception` or `RuntimeException` okay?